wg-deployment/charter.md

```diff
@@ -0,0 +1,51 @@
+# WG Deployment Charter
+
+This charter adheres to the conventions, roles and organization management outlined in [wg-governance].
+
+## Scope
+
+Manage manifests for deploying and operating Kubeflow.
```

Comment:
I'm wondering if there's a way to reword this to make it clear that deploying and managing individual applications is outside the scope of this WG.

What about something along the lines of

> Provide a catalog (centralized repository) of Kubeflow application manifests
> Provide tooling to deploy Kubeflow applications from the catalog

Does this seem more inline with the proposd division of responsibility between this WG and the other WGs?

Reply:
Yeah that makes things clearer :) Thanks for the review !